en/controllers/middleware.rst

```diff
@@ -229,7 +225,9 @@ their own response. We can see both options in our simple middleware::
                     'expire' => '+ 1 year',
                 ]);
             }
-            return $response;
+            
+            // Pass the modified instances to the next Middleware
+            return $next($request, $response);
```

Comment:
Depends on the use case right, sometimes you want to bail early, then you directly return the response before adding more.
But in this case it seems indeed that no bail early is desired.